### PR TITLE
Add episode 22 details in meetup section

### PR DESCRIPTION
**Pull request by kelishrestha**
* Add episode 22 details

https://rubynepal.org/meetup/?episode=22

Remaining to be added:
* ~Youtube video link~
* ~Slides link~
* ~Demo link~

### CODE REVIEW

File: js/meetup/data.js [hunk 1]
```diff
@@ -1,4 +1,47 @@
...
+        title: 'Maintaining Code Standards',
+        description: null,
+        speaker: 'Prabin Poudel',
+        twitter_username: 'coolprobn',
+        github_username: 'coolprobn',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/15196941?v=4',
+        designation: 'Senior Ruby on Rails Engineer @ Truemark Technology',
+        slides_link: '',
+        video_link: '',
```
Comment (josisusan) [inline js/meetup/data.js:40]:
```suggestion
        video_link: 'https://youtu.be/gp9fLd346Pw',
```

File: js/meetup/data.js [hunk 2]
```diff
@@ -1,4 +1,47 @@
...
+        title: 'Hanami 2.0',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
+        github_username: 'timriley',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
+        video_link: '',
```
Comment (josisusan) [inline js/meetup/data.js:26]:
```suggestion
        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
```

File: js/meetup/data.js [hunk 3]
```diff
@@ -1,4 +1,61 @@
...
+        github_username: 'timriley',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: ''
+      },
+      {
+        id: 1,
```
Comment (zoras) [inline js/meetup/data.js:30]:
```suggestion
        id: 2,
```

File: js/meetup/data.js [hunk 4]
```diff
@@ -1,4 +1,61 @@
...
+        github_username: 'timriley',
+        gitlab_username: 'https://youtu.be/gTZHnS9B9Lo',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: ''
+      },
+      {
+        id: 2,
```
Comment (zoras) [inline js/meetup/data.js:44]:
```suggestion
        id: 3,
```

**Review by zoras — CHANGES_REQUESTED** [review]

File: js/meetup/data.js [hunk 5]
```diff
@@ -1,4 +1,61 @@
...
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
+        github_username: 'timriley',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: ''
```
Comment (zoras) [inline js/meetup/data.js:27]:
```suggestion
        demo_link: 'https://github.com/hanami/bookshelf'
```

File: js/meetup/data.js [hunk 6]
```diff
@@ -1,4 +1,61 @@
...
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
+        github_username: 'timriley',
+        gitlab_username: 'https://youtu.be/gTZHnS9B9Lo',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: ''
```
Comment (zoras) [inline js/meetup/data.js:41]:
```suggestion
        demo_link: 'https://github.com/dry-rb/dry-web-blog'
```

File: js/meetup/data.js [hunk 7]
```diff
@@ -1,4 +1,61 @@
...
+        id: 2,
+        title: 'Bonus: Dry-rb',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
+        github_username: 'timriley',
+        gitlab_username: 'https://youtu.be/gTZHnS9B9Lo',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
```
Comment (zoras) [inline js/meetup/data.js:39]:
```suggestion
        slides_link: 'https://dry-rb.org/',
```

File: js/meetup/data.js [hunk 8]
```diff
@@ -1,4 +1,61 @@
...
+        id: 1,
+        title: 'Hanami 2.0',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
+        github_username: 'timriley',
+        gitlab_username: '',
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: '',
```
Comment (zoras) [inline js/meetup/data.js:25]:
```suggestion
        slides_link: 'https://hanamirb.org/',
```

**Review by zoras — CHANGES_REQUESTED** [review]

File: js/meetup/data.js [hunk 9]
```diff
@@ -1,4 +1,61 @@
...
+      twitter_username: 'zoraslapen'
+    },
+    venue_sponsor: {},
+    sessions: [
+      {
+        id: 1,
+        title: 'Hanami 2.0',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
```
Comment (zoras) [inline js/meetup/data.js:20]: Tim moved to mastodon https://ruby.social/@timriley from twitter https://twitter.com/timriley. But we don't link to mastodon currently

```suggestion
        twitter_username: 'timriley',
```

File: js/meetup/data.js [hunk 10]
```diff
@@ -1,4 +1,61 @@
...
+        slides_link: 'https://hanamirb.org/',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: 'https://github.com/hanami/bookshelf'
+      },
+      {
+        id: 2,
+        title: 'Bonus: Dry-rb',
+        description: null,
+        speaker: 'Tim Riley',
+        twitter_username: '',
```
Comment (zoras) [inline js/meetup/data.js:34]: Tim moved to mastodon https://ruby.social/@timriley from twitter https://twitter.com/timriley. But we don't link to mastodon currently

```suggestion
        twitter_username: 'timriley',
```

File: js/meetup/data.js [hunk 11]
```diff
@@ -1,4 +1,61 @@
...
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: 'https://hanamirb.org/',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: 'https://github.com/hanami/bookshelf'
+      },
+      {
+        id: 2,
+        title: 'Bonus: Dry-rb',
+        description: null,
```
Comment (zoras) [inline js/meetup/data.js:32]:
```suggestion
        description: dry-rb is a collection of next-generation Ruby libraries which helps you write clear, flexible, and more maintainable Ruby code.
```

File: js/meetup/data.js [hunk 12]
```diff
@@ -1,4 +1,61 @@
...
+    hosted_by: {
+      name: 'Saroj Maharjan',
+      twitter_username: 'zoraslapen'
+    },
+    venue_sponsor: {},
+    sessions: [
+      {
+        id: 1,
+        title: 'Hanami 2.0',
+        description: null,
```
Comment (zoras) [inline js/meetup/data.js:18]:
```suggestion
        description: Hanami 2: New Framework, New You. Hanami is a modern web framework for Ruby,
```

File: js/meetup/data.js [hunk 13]
```diff
@@ -1,4 +1,61 @@
...
+        external_avatar: 'https://avatars.githubusercontent.com/u/40017038?v=4',
+        designation: 'Principal Engineer @ Buildkite',
+        slides_link: 'https://dry-rb.org/',
+        video_link: 'https://youtu.be/ZqbPUKMSSeg ',
+        demo_link: 'https://github.com/dry-rb/dry-web-blog'
+      },
+      {
+        id: 3,
+        title: 'Maintaining Code Standards',
+        description: null,
```
Comment (zoras) [inline js/meetup/data.js:46]:
```suggestion
        description: Maintaining Coding Standards with RuboCop, Overcommit, Pronto and Gitlab,
```

**Review by zoras — CHANGES_REQUESTED** [review]

File: js/meetup/data.js (hunk 9 above)
Comment (zoras) [reply js/meetup/data.js:20]: I've added support for mastodon link

File: js/meetup/data.js (hunk 10 above)
Comment (zoras) [reply js/meetup/data.js:35]: I've added support for mastodon link

**Review by zoras — APPROVED** [review]
LGTM 💯 

Thanks @josisusan and @kelishrestha

**Pull request merged**